src/ln/msgs.rs

```diff
@@ -382,9 +384,93 @@ impl NetAddress {
 			&NetAddress::OnionV3 {..} => { 4 },
 		}
 	}
+
+	fn len(&self) -> u16 {
```

Comment:
Maybe add doc noting that the type byte isnt counted here?

Reply:
Right, done